README.md

```diff
@@ -121,6 +122,24 @@ ESP-IDF Directory selection dialog:
 
 ![](docs/images/esp_idf_dir.png)
 
+<a name="Installing ESP-IDF and Tools via Tools Manager"></a>
+# Install Tools Manager
+You can use the install tools manager to manage the tools installation via a wizard. The advantage of this method over the exisitng installation is that you can easily manage the whole flow via wizard and install the tools in ESP-IDF framework that you only need.<br/>
```

Comment:
Looks like you've changed the name to "Tools Installation Wizard", we could update here

Reply:
Updated